c/parallel/CMakeLists.txt

```
@@ -34,11 +34,7 @@ add_subdirectory(src/jit_templates)
 
 find_package(CUDAToolkit REQUIRED)
 
```

Add this to use the static runtime library:
```suggestion
set(CMAKE_CUDA_RUNTIME_LIBRARY "STATIC")
```